app/middleware/middleware_healthcheck.rb

```diff
@@ -37,7 +37,6 @@ def call(env)
         if db_healthy && (ENV['launchpad_login_required'] != 'true' || launchpad_healthy)
           response[0] = 200
         end
-        Rails.logger.info "The Status page returned a #{response[0]} Response:#{response.inspect}"
```

Comment:
this looks like it is removing the logging for _both_ a 200 status response as well as a 503 status response.

Reply:
The final log, yes, but there are logs associated with each of the other failure conditions.  If we want to log the failure response here, in addition to those, I could add an else to the preceding if for this logging.